go/vt/topo/etcd2topo/watch.go

```diff
@@ -76,7 +76,7 @@ func (s *Server) Watch(ctx context.Context, filePath string) (*topo.WatchData, <
 		defer close(notifications)
 		defer outerCancel()
 
-		var currVersion = initial.Header.Revision
+		var rev = initial.Header.Revision
```

Comment:
Is there any other actual diff in this file? Everything else looks cosmetic.

Reply:
No, just renaming things to be more precise and clear: https://github.com/vitessio/vitess/pull/15701/files/f1fed6dd787feea26a4f4e13f660e8eaa91e64bb#r1568995668